src/main/java/org/zalando/nakadi/util/Aruha771MigrationHelper.java

```diff
@@ -0,0 +1,63 @@
+package org.zalando.nakadi.util;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.zalando.nakadi.config.JsonConfig;
+import org.zalando.nakadi.domain.Subscription;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.Statement;
+
+/**
+ * Temporary class to be used for DB migration of ARUHA-771
+ */
+public class Aruha771MigrationHelper {
```

Comment:
@adyach @rcillo 
For me the purpose of this class is the same as the purpose of scripts which we put to https://github.com/zalando/nakadi/tree/master/database/migration
It helps to migrate database from one state to another. It is not an SQL-script just because it needs some more complex logic.
Maybe it will make sense to put it not inside nakadi java code but to make a separate source-root for it in this repo. What do you think?